Makefile

```diff
@@ -0,0 +1,18 @@
+all: noembed
+
+submodule:
+	git submodule update --init
+
+noembed: submodule build
+
+generate:
+	go get github.com/jteeuwen/go-bindata
```

Comment:
Oh, I think we need to get github.com/jteeuwen/go-bindata/go-bindata (note the extra go-bindata)